examples/add-filter-processor/rendered_manifests/configmap-agent.yaml

```diff
@@ -19,11 +19,15 @@ data:
   relay: |
     exporters:
       otlphttp:
+        auth:
+          authenticator: headers_setter
         headers:
           X-SF-Token: ${SPLUNK_OBSERVABILITY_ACCESS_TOKEN}
```

Comment:
Why this one is kept?

Reply:
```
headers:
          X-SF-Token: ${SPLUNK_OBSERVABILITY_ACCESS_TOKEN}
```
I went through all components that makes up the passthrough, checkeds previous PRs (standalone config), tested multiple scenarios and I can't see or any reason why we left this.
The only explanation is that the original author for passthrough documented it and left the headers in place and we carried it from there . 
In short, this is redundant and no longer needed when using headers_setter , I will update the PR.
Thanks !